src/mixins/nft.js

```diff
@@ -869,8 +874,8 @@ export default {
     lazyFetchNFTOwners() {
       return this.lazyGetNFTOwners(this.classId);
     },
-    updateNFTOwners() {
-      return this.fetchNFTOwners(this.classId);
+    updateNFTOwners({ nocache = true } = {}) {
```

Comment:
Not sure what should be the default value of `nocache`
if we don't want to change existing behaviour it should be `false`
but if we only call this in non-lazy case, `true` is more sensible